deployment/helm/templates/rbac.yaml

```diff
@@ -90,4 +90,39 @@ subjects:
   - kind: 'ServiceAccount'
     name: 'akri-agent-sa'
     namespace: {{ .Release.Namespace }}
+
+{{- if .Values.cleanupHook.enabled }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: akri-helm-hook-sa
```

Comment:
nit: naming, here and in the clusterRole/clusterRoleBinding - perhaps better to call this `akri-helm-cleanup-hook-sa`